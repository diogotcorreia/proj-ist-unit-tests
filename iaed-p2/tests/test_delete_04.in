set Home/Downloads baixaqui
set users/admin administrador
set users/admin2 backup do administrador
set Home/Desktop/Pictures/Natal-2017 fotos do natal
set Home/Desktop/Homework nothing sus here
print
list
delete //users
list
delete
print
list
quit
