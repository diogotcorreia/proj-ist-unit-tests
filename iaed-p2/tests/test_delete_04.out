/Home/Downloads baixaqui
/Home/Desktop/Pictures/Natal-2017 fotos do natal
/Home/Desktop/Homework nothing sus here
/users/admin administrador
/users/admin2 backup do administrador
Home
users
Home
